views/event-form.hbs

```diff
@@ -0,0 +1,186 @@
+  {{else}}
+    <input type="text" name="cost" required>
+  {{/if}}
+</label>
+
+<label>{{localText.image}}
+  {{#if event.imageUrl}}
+    <input type="text" name="imageUrl" value="{{event.imageUrl}}">
+  {{else}}
+    <input type="text" name="imageUrl">
+  {{/if}}
+</label>
+
+<fieldset>
+  <legend>{{localText.accessibilityOptions}}</legend>
+  <div>
+    <label for="audio-recordings">{{localText.audioRecordings}}</label>
+    <input id="audio-recordings" type="checkbox" name="accessibility" value="Audio recordings" {{checkOptionsValue event.accessibilityOptions "Audio recordings"}}>
+  </div>
+  <div>
+    <label for="braille">{{localText.brailleMenu}}</label>
+    <input id="braille" type="checkbox" name="accessibility" value="Braille" {{checkOptionsValue event.accessibilityOptions "Braille"}}>
+  </div>
+  <div>
+    <label for="big-fonts">{{localText.bigFonts}}</label>
+    <input id="big-fonts" type="checkbox" name="accessibility" value="Big fonts" {{checkOptionsValue event.accessibilityOptions "Big fonts"}}>
+  </div>
+  <div>
+    <label for="carer">{{localText.carer}}</label>
+    <input id="carer" type="checkbox" name="accessibility" value="Carer" {{checkOptionsValue event.accessibilityOptions "Carer"}}>
+  </div>
+  <div>
+    <label for="guide-dog">{{localText.guideDog}}</label>
+    <input id="guide-dog" type="checkbox" name="accessibility" value="Place for guide dog" {{checkOptionsValue event.accessibilityOptions "Place for guide dog"}}>
+  </div>
+  <div>
+    <label for="sms-message">{{localText.smsMessaging}}</label>
+    <input id="sms-message" type="checkbox" name="accessibility" value="SMS messaging" {{checkOptionsValue event.accessibilityOptions "SMS messaging"}}>
+  </div>
+  <div>
+    <label for="sign-lang">{{localText.signLang}}</label>
+    <input id="sign-lang" type="checkbox" name="accessibility" value="Sign language" {{{checkOptionsValue event.accessibilityOptions "Sign language"}}}>
+  </div>
+  <div>
+    <label for="good-lighting">{{localText.goodLighting}}</label>
+    <input id="good-lighting" type="checkbox" name="accessibility" value="Good lighting" {{checkOptionsValue event.accessibilityOptions "Good lighting"}}>
+  </div>
+  <div>
+    <label for="wheelchair">{{localText.wheelchairFriendly}}</label>
+    <input id="wheelchair" type="checkbox" name="accessibility" value="Wheelchair access" {{checkOptionsValue event.accessibilityOptions "Wheelchair access"}}>
+  </div>
+  <div>
+    <label for="disabled-parking">{{localText.disabledParking}}</label>
+    <input id="disabled-parking" type="checkbox" name="accessibility" value="Disabled parking" {{checkOptionsValue event.accessibilityOptions "Disabled parking"}}>
+  </div>
+  <div>
+    <label for="disabled-toilets">{{localText.disabledToilet}}</label>
+    <input id="disabled-toilets" type="checkbox" name="accessibility" value="Disabled toilets" {{checkOptionsValue event.accessibilityOptions "Disabled parking"}}>
```

Comment:
`{{checkOptionsValue event.accessibilityOptions "Disabled parking"}}` should be `{{checkOptionsValue event.accessibilityOptions "Disabled toilets"}}`

Reply:
@Suuuuha do this still please